### PR TITLE
OLS-624: more unit tests for attachment appender

### PR DESCRIPTION
**Pull request by tisnik**
## Description

More unit tests for attachment appender

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)
- [x] Unit tests

## Related Tickets & Documents

- Related Issue #[OLS-624](https://issues.redhat.com//browse/OLS-624)
- Closes #

### CODE REVIEW

File: tests/unit/query_helpers/test_attachment_appender.py [hunk 1]
```diff
@@ -141,6 +161,52 @@ def test_append_two_attachments_to_query(_test_yaml):
     assert output == expected
 
 
+def test_append_incomplete_yaml_query(_yaml_without_kind, _yaml_without_name):
```
Comment (onmete) [inline tests/unit/query_helpers/test_attachment_appender.py:164]: Should those fixtures be underscored? They return and are used.

Comment (tisnik) [reply tests/unit/query_helpers/test_attachment_appender.py:164]: tbh I don't care. If you prefer, I'll change it now

Comment (tisnik) [reply tests/unit/query_helpers/test_attachment_appender.py]: updated @onmete

**Pull request merged**